src/template-tags/tickets.php

```diff
@@ -123,6 +123,11 @@ function tribe_tickets_display_count_and_ticket_button() {
 		// get an array for ticket and rsvp counts
 		$types = Tribe__Tickets__Tickets::get_ticket_counts( $event_id );
 
+		// if no rsvp or tickets return
+		if ( ! $types ) {
+			return null;
```

Comment:
💡 Not introduced as part of this change, so this isn't essential, but I notice the function either echoes a bit of text or else bails out - it doesn't look like it ever returns a string, though (so might be nice to revise or remove the `@return` tag in the docblock).